index.html

```diff
@@ -1820,11 +1820,12 @@ <h2>Algorithms</h2>
 
       <p>
 The algorithms defined below operate on documents represented as <dfn
-data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows
-[[JSON-LD-API]] in representing a JSON object as an [[Infra]] [=map=]. An
-<dfn class="export">unsecured data document</dfn> is a [=map=] with no
-`proof` [=map/key=]. A <dfn class="export">secured data document</dfn> is a
-[=map=] with a `proof` [=map/key=].
+data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows the
+[[[JSON-LD11-API]]] specification in representing a JSON object as an
+[[[Infra]]] [=map=]. An <dfn class="export">unsecured data document</dfn> is a
+[=map=] that has not yet had the desired proof added to it, but it MAY contain a
+proof value added to it by a previous process. A <dfn class="export">secured
+data document</dfn> is a [=map=] that has had the desired proof(s) added to it.
```

Comment:
Instead of "desired" ... since maybe the document already has some desirable proof in it ... perhaps "impending" to identify the proof that is about to be added? Perhaps "target proof" could be used if "impending" is not preferred.

```suggestion
[=map=] that has not yet had the impending proof added to it, but it MAY contain a
proof value added to it by a previous process. A <dfn class="export">secured
data document</dfn> is a [=map=] that has had the impending proof(s) added to it.
```

Reply:
`impending` is good for the first instance, but not for the second, because `impending` suggests an action is yet to be taken, while the second instance is after that action. Perhaps (note that this would also replace line 1825, which really wants to say `In this context` instead of `In this scenario`, but that's guaranteed to confuse our readers) —
```suggestion
[[[Infra]]] [=map=]. In this scenario, an <dfn class="export">unsecured data document</dfn> is a [=map=] that has not yet had the current proof added to it, but it MAY contain a
proof value added to it by a previous process. A <dfn class="export">secured
data document</dfn> is a [=map=] that has had the current proof(s) added to it.
```